### ISSUE TITLE
Tuple built-in cannot be given more than one argument in Python < 3.9 when pyrightconfig.json is present

### ISSUE THREAD

**Issue by layday** [microsoft/pyright#1156]
**Describe the bug**
The built-in `tuple` cannot be subscripted with more than one argument when a `pyrightconfig.json` file is present in versions of Python lower than 3.9.

**To Reproduce**
Create a test module containing only:

```py
from __future__ import annotations

foo: tuple[str, str] = ('abc', 'def')
bar: tuple[str] = ('ghi',)  # works
```

and invoke Pyright, first without and then with a `pyrightconfig.json`:

```
$ npx pyright --verbose
npx: installed 1 in 1.561s
No configuration file found.
Search paths found for configured python interpreter:
  /nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python3.8/site-packages
  /nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python3.8
  /nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python3.8/lib-dynload
When attempting to get search paths from python interpreter:
  Executing interpreter: 'python3'
  Skipping '/nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python38.zip' because it is not a valid directory
  Received 3 paths from interpreter
    /nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python3.8/site-packages
    /nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python3.8
    /nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python3.8/lib-dynload
stubPath /[...]/test/typings is not a valid directory.
Assuming Python platform Darwin
Searching for source files
Found 1 source file
0 errors, 0 warnings, 0 infos
Completed in 0.617sec

$ echo '{}' > pyrightconfig.json

$ npx pyright --verbose
npx: installed 1 in 1.505s
Loading configuration file at /[...]/test/pyrightconfig.json
Assuming Python version 3.8
Assuming Python platform Darwin
No include entries specified; assuming /[...]/test
Auto-excluding **/node_modules
Auto-excluding **/__pycache__
Auto-excluding .git
Search paths found for configured python interpreter:
  /nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python3.8/site-packages
  /nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python3.8
  /nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python3.8/lib-dynload
When attempting to get search paths from python interpreter:
  Executing interpreter: 'python3'
  Skipping '/nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python38.zip' because it is not a valid directory
  Received 3 paths from interpreter
    /nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python3.8/site-packages
    /nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python3.8
    /nix/store/0wkqz5gy7x2sxx02l402l8dpgbn5zq37-python3-3.8.6/lib/python3.8/lib-dynload
stubPath /[...]/test/typings is not a valid directory.
Searching for source files
Found 1 source file
/[...]/test/test.py
  3:17 - error: Too many type arguments provided for class "tuple"; expected 1 but received 2 (reportGeneralTypeIssues)
1 error, 0 warnings, 0 infos
Completed in 0.627sec

$ python3 -V
Python 3.8.6
```

**Expected behavior**
That `tuple` would be variadic as normal.

**VS Code extension or command-line**
Either, 1.1.84

**Comment by erictraut** [comment]
Thanks, this will be fixed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.85, which I just published. It will also be included in the next release of pylance.
